src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Relic-Strings.json

```diff
@@ -6,6 +6,16 @@
       "Reduces stsjorbsmod:Burning falloff rate to 10% per turn."
     ]
   },
+  "stsjorbsmod:BottledBurdenRelic": {
+    "NAME": "Bottled Burden",
+    "FLAVOR": "I carry this with me, always",
+    "DESCRIPTIONS": [
+      "Upon pickup, choose a card that does not #yExert. Start each combat with that card #yEntombed. #yExhume it on turn 3.",
+      "Choose a card for ",
+      "At the start of turn 3 ",
```

Comment:
Would be a little more localization friendly for this to be one string with a placeholder rather than 2 strings appended together (ie, `At the start of turn %1$s, Exhume %2$s.`) - the existing strings aren't perfect about this, but we try to be better with newer strings